test/spmd/tensor/test_device_mesh.py

```diff
@@ -422,6 +423,33 @@ def test_scatter_nd(self):
             received_tensor = mesh.scatter(scattered_tensors, mesh_dim=dim)
             self.assertEqual(received_tensor, torch.ones(3, 3) * self.rank)
 
+    @with_comms
+    @skip_if_lt_x_gpu(8)  # meaning self.world_size but self is not in scope
+    def test_all_to_all_1d(self):
```

Comment:
could you also add a nd all_to_all test similar to others?

Reply:
Yes. I was about to do that in a follow up PR because I thought that supporting n-d broadcast needed to change my `all_to_all` implementation but actually it seems no. So I can just add n-d test without changing the ops code.